cas/ut/client_ut.c

```diff
@@ -90,7 +91,10 @@ static char *cas_startup_cmd[] = { "m0d", "-T", "linux",
 static const char         *cdbnames[] = { "cas1" };
 static const char      *cl_ep_addrs[] = { "0@lo:12345:34:2" };
 static const char     *srv_ep_addrs[] = { "0@lo:12345:34:1" };
-static struct m0_net_xprt *cs_xprts[] = { &m0_net_lnet_xprt };
+static struct m0_net_xprt *cs_xprts[] = { &m0_net_lnet_xprt,
+					  &m0_net_sock_xprt
+					  /*&m0_net_libfabric_xprt*/
+					};
```

Comment:
Now, I hope you can find a way to not directly use these xprt.

They all should be registered into a list.
Or they should be defined somewhere, and be accessed in other places.

Please refer to:
```
 55 static struct {
 56         const char         *name;
 57         struct m0_net_xprt *xprt;
 58 } net_xprt_mods[] = {
 59         [M0_NET_XPRT_LNET] = {
 60                 .name = "\"lnet\" m0_net_xprt_module",
 61                 .xprt = &m0_net_lnet_xprt
 62         },
 63         [M0_NET_XPRT_BULKMEM] = {
 64                 .name = "\"bulk-mem\" m0_net_xprt_module",
 65                 .xprt = &m0_net_bulk_mem_xprt
 66         }
"net/module.c" 140 lines --20%--                 
```
Can we use this in all other places?